cedar-policy-core/src/evaluator.rs

```diff
@@ -172,7 +172,7 @@ impl<'e> RestrictedEvaluator<'e> {
                 match split(args) {
                     Either::Left(values) => {
                         let values : Vec<_> = values.collect();
-                        let efunc = self.extensions.func(fn_name).map_err(|err| err.with_maybe_source_loc(expr.source_loc().cloned()))?;
+                        let efunc = self.extensions.func(fn_name)?;
```

Comment:
why this change?

Reply:
`Name`s carry a source location, so I update the error to use that